code/datums/components/crafting/crafting.dm

```diff
@@ -549,6 +562,8 @@
 	// Crafting
 	if(recipe.non_craftable)
 		data["non_craftable"] = recipe.non_craftable
+	if(recipe.mass_craftable)
+		data["mass_craftable"] = recipe.mass_craftable
```

Comment:
```suggestion
	data["mass_craftable"] = recipe.mass_craftable
```
Checking if this value is `true` is redundant - If it's `false`, we pass `mass_craftable` as `null`, which ends up being the same thing as passing it as `false`.